.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+## PR Checklist
+Please check if your PR fulfills the following requirements:
+
+- [ ] The commit message follows our guidelines: https://github.com/okta/okta-oidc-js/blob/master/CONTRIBUTING.md#commit
```

Comment:
This can be a relative link: [guidelines](/okta/okta-oidc-js/CONTRIBUTING.md#commit)